modal/cli/app.py

```diff
@@ -63,10 +63,11 @@ async def run(
             f"No function `{function_name}` could be found in the specified stub. Registered functions are: {registered_functions_str}"
         )
         exit(1)
-
-    async with _stub.run(detach=detach) as app:
+    blocking_stub = synchronizer._translate_out(_stub, Interface.BLOCKING)
+    with blocking_stub.run(detach=detach) as app:
         func_handle = getattr(app, function_name)
-        await func_handle.call()
+        blocking_function = synchronizer._translate_out(synchronizer._translate_in(func_handle), Interface.BLOCKING)
```

Comment:
Not sure if we have to do the translation stuff for the function handle as well or if the blocking stub is smart enough to always return blocking version of FunctionHandle when doing getattr? @erikbern

Reply:
i think this looks right!